### PR TITLE
Release 5.0.0

### PR DESCRIPTION
**Pull request by cmurphy**

### CODE REVIEW

**Comment by cmurphy** [comment]
This should not be merged until all the other mergeable PRs in the queue are merged.

File: README.md [hunk 1]
```diff
@@ -87,9 +88,9 @@ Additionally, you need to know the network address ranges for all four of the pu
 and the specific ip addresses of the controller node and the storage node. Keep in mind that your
 public networks can overlap with one another, as can the private networks.
 
-If you are running VMWare Fusion, and Vagrant with the Fusion provider, a CentOS 6.5 image is provided
-to help you get started. It is stored in the Vagrant Cloud, and will be downloaded automatically.
-See the examples/multinode and examples/allinone directories for details.
+If you are running VMWare Fusion, and Vagrant with the Fusion provider, a CentOS 7 image is given by
```
Comment (cyberious) [inline README.md]: perhaps reworded to 'CentOS 7 Vagrantfile provided'...

File: metadata.json [hunk 2]
```diff
@@ -23,24 +23,25 @@
...
 
-    { "name": "puppetlabs/ntp", 	"version_requirement": ">=3.0.0 <4.0.0" },
-    { "name": "puppetlabs/firewall", 	"version_requirement": ">=1.0.0 <2.0.0" },
-    { "name": "puppetlabs/vswitch", 	"version_requirement": ">=0.2.0 <1.0.0" },
-    { "name": "puppetlabs/mongodb", 	"version_requirement": ">=0.6.0 <1.0.0" },
-    { "name": "puppetlabs/mysql", 	"version_requirement": ">=2.2.0 <3.0.0" },
-    { "name": "puppetlabs/rabbitmq", 	"version_requirement": ">=3.0.0 <4.0.0" },
-    { "name": "puppetlabs/apache", 	"version_requirement": ">=1.0.0 <1.2.0" },
+    { "name": "puppetlabs/apache", "version_requirement": ">=1.2.0 <2.0.0" },
+    { "name": "puppetlabs/firewall", "version_requirement": ">=1.0.0 <2.0.0" },
```
Comment (cyberious) [inline metadata.json]: these can be shorthanded by use 1.x according to the Forge Team

Comment (cmurphy) [inline metadata.json]: The stackforge modules us this format so I'd like to keep it for consistency with them.

**Pull request merged**